src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/RegisterChecks/Orchestrator/RoepaoTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Infrastructure;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Tests.Services.Gateway.RegisterChecks.Orchestrator
+{
+    [TestFixture]
+    public class RoepaoTests
+    {
+        private GatewayRegisterChecksOrchestrator _orchestrator;
+        private Mock<IRoatpApplicationApiClient> _applyApiClient;
+        private Mock<IRoatpApiClient> _roatpApiClient;
+        private Mock<ILogger<GatewayRegisterChecksOrchestrator>> _logger;
+
+        private static string ukprn = "12345678";
+        private static string UKRLPLegalName = "John's workshop";
+        private static string UserName = "GatewayUser";
+
+        private readonly Guid _applicationId = Guid.NewGuid();
+
+        [SetUp]
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _roatpApiClient = new Mock<IRoatpApiClient>();
+            _logger = new Mock<ILogger<GatewayRegisterChecksOrchestrator>>();
+            _orchestrator = new GatewayRegisterChecksOrchestrator(_applyApiClient.Object, _roatpApiClient.Object, _logger.Object);
+
+            var commonDetails = new GatewayCommonDetails
+            {
+                ApplicationSubmittedOn = DateTime.Now.AddDays(-3),
+                CheckedOn = DateTime.Now,
+                LegalName = UKRLPLegalName,
+                Ukprn = ukprn
+            };
+            _applyApiClient.Setup(x => x.GetPageCommonDetails(It.IsAny<Guid>(), It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync(commonDetails);
```

Comment:
Can you replace IsAny with specific values?

Reply:
done